drivers/include/lis2dh12.h

```diff
@@ -194,6 +207,7 @@ typedef struct {
  */
 extern const saul_driver_t lis2dh12_saul_driver;
 
+#if MODULE_LIS2DH12_INT || DOXYGEN
```

Comment:
```suggestion
#if defined(MODULE_LIS2DH12_INT) || DOXYGEN
```

Reply:
This change is not necessary. A not defined name is evaluated to 0 by the preprocessor.

I personally prefer not to use `defined` in such cases because it can cause long cascades of `defined()` if you use a larger number of such tests which makes it not easier to read.

BTW, `DOXYGEN` is use in same way.

Reply:
Also in case of `MODULE_*` definitions, we already use it without `defined`, for example:
```
./cpu/cortexm_common/vectors_cortexm.c
./drivers/lpsxxx/lpsxxx.c
./drivers/netdev_ieee802154/netdev_ieee802154.c
./drivers/include/saul/periph.h
...
```